Framework/Kernel/inc/MantidKernel/Material.h

```diff
@@ -50,21 +50,36 @@ File change history is stored at: <https://github.com/mantidproject/mantid>
 */
 class MANTID_KERNEL_DLL Material {
 public:
+  /// Structure to hold the information for a parsed chemical formula
+  struct FormulaUnit {
+    boost::shared_ptr<PhysicalConstants::Atom> atom;
```

Comment:
Should we try and avoid the `shared_ptr` overhead with `unique_ptr` or do we want shared semantics?